### DISCUSSION TITLE
Adding other cards to floor plan cards!

### DISCUSSION THREAD

**Discussion by Trzinka** [ExperienceLovelace/ha-floorplan#428] [Hanging feature requests]
## Hi!

# Are you thinking in the direction of adding other cards to the floor plan card.

**The floor plan card itself displays beautifully on mobile devices. The problem arises if I want to display something else.**

`Example of card:`

![bitmap](https://github.com/ExperienceLovelace/ha-floorplan/assets/40424965/1898107f-9b28-4c6b-bfd5-fb6070a19f20)

`The display mode on the tablet can still be adjusted somehow:`

![Screenshot_20240214-181541](https://github.com/ExperienceLovelace/ha-floorplan/assets/40424965/781c881b-2c62-4575-b88d-cac090fe76bf)

`It's a completely different story on a mobile phone:`

![Mobitel_Home Assistant](https://github.com/ExperienceLovelace/ha-floorplan/assets/40424965/479560bc-4bd2-46f7-9606-8709e046a62a)

# With the help of OzGav https://github.com/ExperienceLovelace/ha-floorplan/discussions/427 we came up with some kind of solution for which I don't know if it will work as I want and there is still a lot of work to do with the preparation of the SVG file. Thet I'm not even talking about the later implementation in the yaml file!!

# Therefore, allowing other cards to be added to the floor plan card would save a lot of work.

**Comment by OzGav** [comment]
It is not a question of "allowing". It just isn't possible as floorplan works by manipulating a SVG file. You can't embed other things into a SVG at runtime.

**Reply by Trzinka** [reply]
# Hi, sorry maybe I mispronounced allow/enable, but you got what i meant! 

**I started as you advised me and ran into the fact that making the SVG file would take a lot of time, especially since it's a try/fix thing, so I was wondering if there was any thought in the direction of enabling other cards.** 

`Thank you for the explanation.`
